CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -94,11 +95,13 @@ class HomeViewController: UIViewController {
     }
 
     private func setupConstraints() {
-        let padding = 35
+        let padding: CGFloat = 70
+        let logoutPadding: CGFloat = LayoutHelper.shared.setCustomVerticalPadding(size: 30) // TODO: Not sure about dimensions.
+        let reachOutPadding: CGFloat = LayoutHelper.shared.setCustomVerticalPadding(size: 70) // TODO: Not sure about dimensions.
```

Comment:
I feel `reachOutButton` doesn't need to be so much above the bottom because the `logoutButton` is going to be removed later

Reply:
Oh ya, these padding values are pretty arbitrary for now because I didn't see the match screen on Zeplin and in general, I think we have to do some more work to see what is the best spacing values are to look good on all screens. I chose these values for now cause I thought they looked good on both iPhone X + iPhone 8. 

![Uploading Screen Shot 2020-02-20 at 9.49.41 PM.png…
<img width="510" alt="Screen Shot 2020-02-20 at 9 48 27 PM" src="https://user-images.githubusercontent.com/29307883/74999960-0b695200-542b-11ea-93cc-3c89ad8774e0.png">
]()